README

```diff
@@ -0,0 +1,5 @@
+# Apollo Client Devtools
+
+This package is published so that some modules can be imported by other devtools.
```

Comment:
Maybe a slight wording adjustment here (so we're explicit):

```suggestion
This package is published so that some of its modules can be imported by other devtools.
```